src/test/scala/com/github/music/of/the/ainur/almaren/jdbc/Test.scala

```diff
@@ -6,148 +6,147 @@ import org.apache.spark.sql.functions._
 import com.github.music.of.the.ainur.almaren.Almaren
 import com.github.music.of.the.ainur.almaren.builder.Core.Implicit
 import com.github.music.of.the.ainur.almaren.jdbc.JDBC.JDBCImplicit
+import org.scalatest.funsuite.AnyFunSuite
```

Comment:
I think existing test.scala file in spark 2.4 will work only this change as to be done at spark 3.3 version